### DISCUSSION TITLE
What graphics i should use?

### DISCUSSION THREAD

**Discussion by Shlepsss** [CosmosOS/Cosmos#2912] [Q&A]
Cosmos have too much ways implementing a graphics, so what i should use if i want to start my OS on real hardware and virtual machines. VBE, VGA, Canvas or SVGAII?

**Comment by valentinbreiz** [comment]
VBE for high res graphics and VGA for larger support but low res

**Comment by Shlepsss** [comment]
> VBE for high res graphics and VGA for larger support but low res

Can i start my OS with VGACanvas on real hardware with GTX 660 and HDMI connect type? Or connect type doesn't matter @valentinbreiz
